src/commands.ts

```diff
@@ -712,10 +712,14 @@ export const insertGraphicalAbstract = (
   dispatch?: Dispatch,
   view?: EditorView
 ) => {
-  // check if another graphical abstract already exists
-  // parameter 'deep' must equal true to search the whole document
+  const GraphicalAbstractSectionNode = findChildrenByType(
+    state.doc,
+    schema.nodes.graphical_abstract_section
+  )[0]
```

Comment:
that will throw an error if there is no GA section! `getChildOfType` with your `!isRejectedInsert` should work for all cases

Reply:
It doesn't throw any exceptions. I've already tested it in all scenarios: empty document, rejected cases, and no GA section. We're using the same method on line 760 to retrieve the abstract section, even if it doesn't exist in the document and other places inside command.ts. Additionally, I need it to be passed inside !isRejectedInsert.

Reply:
done. please check